lib/quandl/model/dataset.rb

```diff
@@ -0,0 +1,20 @@
+module Quandl
+  class Dataset < ModelBase
+    include Quandl::Operations::Get
+    include Quandl::Operations::List
+
+    def database
+      Quandl::Database.get(database_code)
+    end
+
+    def data(options = {})
+      Quandl::Data.all({ params: { database_code: database_code, dataset_code: dataset_code } }.deep_merge(options))
+    end
+
```

Comment:
make sure `column_names` does not use base.rb's implementation but actually outputs the attribute returned